schemas/zgrab2.py

```diff
@@ -20,7 +20,8 @@ def DebugOnly(childType):
 
 # zgrab2/processing.go: Grab
 grab_result = Record({
-    "ip": IPv4Address(required = True),
+    "ip": IPv4Address(required = False),
```

Comment:
This is crazy nit-picky, but generally named params don't get spaces in Python per PEP8. So, you probably want `"ip": IPv4Address(required=False),`

Reply:
Ran this through the PEP8 linter...should be good now.